planner/core/exhaust_physical_plans.go

```diff
@@ -959,6 +959,8 @@ func (p *LogicalJoin) constructInnerTableScanTask(
 		isPartition:     ds.isPartition,
 
 		underInnerIndexJoin: true,
+		tblCols:             ds.TblCols,
+		tblColHists:         ds.TblColHists,
```

Comment:
We use `ts.StoreType` but we don't explicitly set it(just implicitly set it to `TiKV`). Do we need to improve it in the future?

Reply:
Yes, no need to set it explicitly here since `IndexLookJoin` is only for `TiKV`.